compiler/src/dmd/expression.d

```diff
@@ -2172,11 +2172,6 @@ extern (C++) final class VoidInitExp : Expression
         this.type = var.type;
     }
 
-    override const(char)* toChars() const
-    {
-        return "void";
-    }
-
```

Comment:
Please update `dmd/ctfe.h`  (no idea why `cafe.h` is where `VoidInitExp` is declared)

Reply:
done